source/manuals/working-with-aws-accounts.html.md.erb

```diff
@@ -0,0 +1,346 @@
+---
+title: Working with AWS accounts
+last_reviewed_on: 2023-11-24
+review_in: 12 months
+---
+
+# <%= current_page.data.title %>
+
+Most teams in GDS use [Amazon Web Services (AWS)](https://aws.amazon.com/) as their infrastructure provider. GDS teams in GOV.UK and DSP manage their own AWS accounts, but users must first sign into a shared base AWS account called `gds-users`. They can then assume roles in their team's AWS account to perform administrative tasks using [AWS's cross-account access pattern](https://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_cross-account-with-roles.html).
+
+Note GOV.UK One Login has its own AWS organisation separate from GDS / Cabinet Office accounts. That is managed by the program and separate guidance applies.
```

Comment:
```suggestion
Note that GOV.UK One Login has its own AWS Organization separate from GDS / Cabinet Office accounts. That is managed by the programme, and separate guidance applies.
```

Feeling quite smug about a suggestion that swaps words between US and British English in both directions in the same paragraph :-D